content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -163,6 +163,48 @@ When reviewing, use the following as a starting point.
+- Before reviewing blog PRs, be familiar with [Submitting blog posts and case studies](/docs/contribute/new-content/blogs-case-studies/).
+- We are willing to mirror any blog article that was published to https://kubernetes.dev/blog/ (the contributor blog) provided that:
+- the mirrored article has the same publication date as the original (it should have the same publication time too, but you can also set a time stamp up to 12 hours later for special cases)
+  - for PRs that arrive the original article was merged to https://kubernetes.dev/, there haven't been 
+  (and won't be) any articles published to the main blog between time that the original and mirrored article 
+  [will] publish. 
+  This is because we don't want to add articles to people's feeds, such as RSS, except at the very end of their feed.
+  - the original article doesn't contravene any strongly recommended review guidelines or community norms.
+  - You should set the canonical URL for the mirrored article, to the URL of the original article 
+  (you can use a preview to predict the URL and fill this in ahead of actual publication). Use the `canonicalUrl` 
+  field in [front matter](https://gohugo.io/content-management/front-matter/) for this.
+- Consider the target audience and whether the blog post is appropriate for kubernetes.io 
+  For example, if the target audience are Kubernetes contributors only then kubernetes.dev
+  may be more appropriate, 
+  or if the blog post is on general platform engineering then it may be more suitable on another site.
+
+  This consideration applies to mirrored articles too; although we are willing to consider all valid
+  contributor articles for mirroring if a PR is opened, we don't mirror all of them.
+
+- We only mark blog articles as maintained (`evergreen: true` in front matter) if the Kubernetes project 
+  is happy to commit to maintaining them indefinitely. Some blog articles absolutely merit this, and we 
+  always mark our release announcements evergreen. Check with other contributors if you are not sure 
+  how to review on this point.
+- The [content guide](/docs/contribute/style/content-guide/) applies unconditionally to blog articles
+  and the PRs that add them. Bear in mind that some restrictions in the guide state that they are only relevant to documentation; those restrictions don't apply to blog articles.
+- The [style guide](/docs/contribute/style/style-guide/) largely also applies to blog PRs, but we make some exceptions.
+  
+  - it is OK to use “we“ in a blog article that has multiple authors, or where the article introduction clearly indicates that the author is writing on behalf of a specific group.
+  - we avoid using Kubernetes shortcodes for callouts (such as `{{</* caution */>}}`)
+  - statements about the future are OK, albeit we use them with care in official announcements on
+    behalf of Kubernetes
+  - code samples don't need to use the `{{< code_sample >}}` shortcode, and often it is better if they do not
```

Comment:
The build preview has failed again with the following error [(logs here)](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/671e7d918ede570008dba59f):

> ERROR [en] "" not found in "docs/contribute/review"

Similar to the previous error. Suggested fix below.
_(In the final rendered page, it will display as intended,`{{< code_sample >}}`)._

```suggestion
  - code samples don't need to use the `{{</* code_sample */>}}` shortcode, and often it is better if they do not
```